native/Avalonia.Native/src/OSX/Screens.mm

```diff
@@ -1,36 +1,59 @@
 #include "common.h"
+#include "AvnString.h"
 
 class Screens : public ComSingleObject<IAvnScreens, &IID_IAvnScreens>
 {
-    public:
+public:
     FORWARD_IUNKNOWN()
+
+    Screens(IAvnScreenEvents* events) {
+        CGDisplayRegisterReconfigurationCallback(CGDisplayReconfigurationCallBack, events);
```

Comment:
Yeah, I was thinking about that. But not sure if it's needed, since CCW will be alive until managed side holds it, right?

Although now I am thinking that in this corner case, when managed side was destroyed, but CGDisplayRegisterReconfigurationCallback was still executed, this pointer will point into something undefined.

Reply:
Done